### PR TITLE
removed package lock from .gitnore, Bumped node-fetch for DOS vuln

### PR DESCRIPTION
**Pull request by kairb**
This change is [Reviewable](https://reviewable.io/reviews/xcov19/mycovidconnect/97)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -102,6 +102,4 @@ dist
 .dynamodb/
 
 # TernJS port file
-.tern-port
-
-package-lock.json
+.tern-port
```
Comment (codecakes) [inline .gitignore:105]: add an EoF blank line

**Review by codecakes — CHANGES_REQUESTED** [review]
@kairb  pls make the changes and get a fresh merge from master branch and then commit back. rest looks good.

**Comment by codecakes** [comment]
@Jasonwei1106 can u look into this and take this up? take a fresh merge from master and push commits?

**Review by ksr89 — APPROVED** [review]

**Review by codecakes — APPROVED** [review]
pls merge and add ur name under credits in README

**Comment by kairb** [comment]
> pls merge and add ur name under credits in README

Please when ready as I don't have permissions

**Comment by codecakes** [comment]
> > pls merge and add ur name under credits in README
> 
> Please when ready as I don't have permissions

pls take a fresh merge. you should be able to merge after that

**Pull request closed without merging**